Simplified/it.lproj/Localizable.strings

```diff
@@ -143,6 +143,10 @@
 "Next Chapter" = "Capitolo successivo";
 
 // NYPLReaderSettingsView
+"Special Font" = "Font speciale";
+"Sans" = "Sans";
+"Serif" = "Serif";
+"Open dyslexic" = "Font per dislessia";
```

Comment:
I think Xcode wants the same capitalization for the key, so `"Open Dyslexic"` which is what you used elsewhere if I'm not mistaken

Reply:
Nice catch, thanks